tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -280,31 +278,25 @@ def test_positive_update_2(self, test_data):
         @Assert: Lifecycle environment description is updated
 
         """
-
-        payload = {
+        new_obj = make_lifecycle_environment({
             'organization-id': self.org['id'],
-        }
-
-        new_obj = make_lifecycle_environment(payload)
+        })
 
         # Update its description
-        result = LifecycleEnvironment.update(
-            {
-                'organization-id': self.org['id'],
-                'id': new_obj['id'],
-                'description': test_data['description'],
-            }
-        )
+        result = LifecycleEnvironment.update({
+            'description': test_data['description'],
+            'id': new_obj['id'],
+            'organization-id': self.org['id'],
+            'prior': new_obj['prior-lifecycle-environment'],
+        })
         self.assertEqual(result.return_code, 0)
         self.assertEqual(len(result.stderr), 0)
```

Comment:
I'm not in favor of removing this, the reason is here https://github.com/SatelliteQE/robottelo/pull/2178#discussion_r29776579